auto_sizing/size_calculation.py

```diff
@@ -149,7 +147,6 @@ def run(self, current_date: datetime) -> None:
 
         if len(metrics_table) == 0:
             print("No clients satisfied targeting.")
-            return 0
```

Comment:
So it looks like this will still work if we continue, but I'm guessing the purpose of `return 0` was to just stop execution, in which case I'd suggest an empty `return` to exit the function early.